src/Command/Initializer/Profiles/blank/Init.php

```diff
@@ -0,0 +1,542 @@
+
+        foreach ($filesToCreate as $file) {
+            if (!is_file($directoryFullPath . DIRECTORY_SEPARATOR . $file)) {
+                touch($directoryFullPath . DIRECTORY_SEPARATOR . $file);
+            }
+        }
+
+        if ($this->useDefaultSetup ? true : $this->io->confirm('Do you want to create docker-compose.yml?', true)) {
+            copy(
+                $this->templatesDirectory . DIRECTORY_SEPARATOR . 'docker-compose.yml.template',
+                $directoryFullPath . DIRECTORY_SEPARATOR . 'docker-compose.yml'
+            );
+        }
+
+        copy(
+            $this->templatesDirectory . DIRECTORY_SEPARATOR . '.gitignore.template',
+            $directoryFullPath . DIRECTORY_SEPARATOR . '.gitignore'
+        );
+        copy(
+            $this->templatesDirectory . DIRECTORY_SEPARATOR . 'public' . DIRECTORY_SEPARATOR . '.htaccess.template',
+            $directoryFullPath . DIRECTORY_SEPARATOR . $directoriesToCreate['index'] . DIRECTORY_SEPARATOR . '.htaccess'
+        );
+
+        // Setup PHPUnit.
+
+        copy(
+            $this->templatesDirectory . DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR . 'bootstrap.php.template',
+            $directoryFullPath . DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR . 'bootstrap.php'
+        );
+
+        $phpunitTemplate = file_get_contents($this->templatesDirectory . DIRECTORY_SEPARATOR . 'phpunit.xml.template');
+        $phpunitTemplate = str_replace(
+            ['{testsDirectory}', '{sourceDirectory}'],
+            [
+                '.' . DIRECTORY_SEPARATOR . ($this->config ? $this->config->getSourceDir() : 'src')
+                . DIRECTORY_SEPARATOR,
+                '.' . DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR,
+            ],
+            $phpunitTemplate ? $phpunitTemplate : ''
+        );
+        file_put_contents($directoryFullPath . DIRECTORY_SEPARATOR . 'phpunit.xml', $phpunitTemplate);
+
+        $composerJsonContent['require-dev']['phpunit/phpunit'] = Versions::PHP_UNIT;
+        $composerJsonContent['scripts']['test'] = 'phpunit';
+        $composerJsonContent['autoload-dev'] = [
+            'psr-4' => [
+                'Tests\\' => 'tests',
+            ],
+        ];
+
+        // End of Setup PHPUnit.
+
+        return $this->writeToComposerJson($directoryFullPath, $composerJsonContent);
+    }
+
+    /**
+     * Setup project dependencies.
+     *
+     * @param string $projectDirectory
+     *
+     * @return int The Exit Code.
+     */
+    protected function setupDependencies(string $projectDirectory): int
+    {
+        $directoryFullPath = getcwd() . DIRECTORY_SEPARATOR . $projectDirectory;
+        $composerJsonContent = $this->readComposerJson($directoryFullPath);
+        $bootstrapDirectory = $this->config ? $this->config->getBootstrapDir() : 'app';
+        $indexDirectory = $this->config ? $this->config->getIndexDir() : 'public';
+        $dependencies = $this->askDependencies();
+        $templates = [
```

Comment:
Should we make this an object that manages all these attributes instead? Accessing arrays `['property']` everywhere to me isn't very maintainable.

Reply:
@l0gicgate what do You think about this [commit](https://github.com/slimphp/Slim-Console/pull/22/commits/1347dedbc734cd85b02671c85c7d62044874ec26)? I moved this logic to a new `FileBuilder` class.